spec/index.bs

```diff
@@ -6126,7 +6130,7 @@ dictionary GPURenderPassDescriptor : GPUObjectDescriptorBase {
 
     : <dfn>occlusionQuerySet</dfn>
     ::
-        Issue: Describe this dictionary member
+        The {{GPUQuerySet}} value defines where the occlusion query results will be output when beginning occlusion query.
```

Comment:
let's reword this a bit. I find "when beginning" a bit confusing here since the occlusion query may cover the whole pass.
 Maybe "defines where the occlusion query results will be stored for this pass`?